web/portal/client/src/entities/User/Action/Impersonate.tsx

```diff
@@ -0,0 +1,46 @@
+import { ActionItemProps } from '@irontec/ivoz-ui';
+import { MoreMenuItem } from '@irontec/ivoz-ui/components/List/Content/Shared/MoreChildEntityLinks';
+import { StyledTableRowChildEntityLink } from '@irontec/ivoz-ui/components/List/Content/Table/ContentTable.styles';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import { useStoreState } from 'store';
+
+const Impersonate = (props: ActionItemProps) => {
+  const { row, entityService, variant = 'icon' } = props;
+  const { email } = row;
+  const token = useStoreState((state) => state.auth.token);
+  const queryString = `token=${token}&email=${email}`;
+  const profile = useStoreState((state) => state.clientSession.aboutMe.profile);
+  const isClientVpbx = profile?.vpbx !== true;
+
+  const handleClickOpen = (e: React.MouseEvent) => {
+    if (isClientVpbx) {
+      e.preventDefault();
+
+      return;
+    }
+  };
+
+  return (
+    <span
+      className={isClientVpbx ? 'disabled' : ''}
+      onClick={(e) => handleClickOpen(e)}
```

Comment:
Only vpbx clients have users and capturing click events is not how we use to disable a button. 
Just remove this.